policies/policy_service.go

```diff
@@ -202,3 +202,37 @@ func validatePolicyBackend(p *Policy, format string, policyData string) (err err
 	}
 	return nil
 }
+
+func (s policiesService) ValidatePolicy(ctx context.Context, token string, p Policy, format string, policyData string) (Policy, error) {
+
+	mfOwnerID, err := s.identify(token)
+	if err != nil {
+		return Policy{}, err
+	}
+
+	if !backend.HaveBackend(p.Backend) {
```

Comment:
The code between the line 213 and 230 was extracted to a function called validatePolicyBackend, please check on policy_service file and the functions.

You can use with the example below
```err = validatePolicyBackend(p, format, policyData)```